### PR TITLE
Convert `pluginSetupWizard.less` to SCSS

### PR DESCRIPTION
**Pull request by janfaracik**
This PR converts 'pluginSetupWizard.less' to SCSS, allowing us to remove LESS dependencies. There's a fair few changes so I've documented them individually as part of this PR.

The eventual plan is to remove Bootstrap from the setup wizard (and eventually from Jenkins), it's quite a heavy library and we're not really taking advantage of it in the setup wizard, nor does it fit in with the ongoing design work on Jenkins.

### Testing done

Screenshot 2023-05-12 at 19 31 10
Screenshot 2023-05-12 at 19 31 21
Screenshot 2023-05-12 at 19 31 25
Screenshot 2023-05-12 at 19 31 31
Screenshot 2023-05-12 at 19 33 23
Screenshot 2023-05-12 at 19 33 26
Screenshot 2023-05-12 at 19 33 31

* There should be no visual differences to the setup wizard

### Proposed changelog entries

- N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7961"></a>

### CODE REVIEW

File: .gitattributes
Comment (janfaracik) [inline .gitattributes:18]: Removed this as there's no longer any `.less` files in Jenkins

File: war/.prettierignore
Comment (janfaracik) [inline war/.prettierignore:35]: Removed this as there's no longer any .less files in Jenkins

File: war/.stylelintrc.js
Comment (janfaracik) [inline war/.stylelintrc.js:4]: We don't to lint the minified version of Bootstrap.

File: war/src/main/scss/pluginSetupWizard.scss
Comment (janfaracik) [inline war/src/main/scss/pluginSetupWizard.scss:7]: Had to move these out as mixins inside of mixins aren't allowed

File: war/webpack.config.js
Comment (janfaracik) [inline war/webpack.config.js:67]: Unsure if this is needed - all the fonts in the fonts folder have been removed.

File: war/src/main/scss/_bootstrap.scss
Comment (janfaracik) [inline war/src/main/scss/_bootstrap.scss:1]: OG plan was to only include the classes used in the wizard, but there's too many and it was too time consuming.

File: .gitattributes
Comment (janfaracik) [reply .gitattributes:18]: Could this be an issue for `.scss` files?

File: .github/renovate.json
Comment (janfaracik) [inline .github/renovate.json:17]: Bootstrap has been moved into its own SCSS file (just for the setup wizard), where it can eventually be removed.

**Review by timja — APPROVED** [review]
Tested manually, looks great.

I'm confused why bootstrap was a dependency and checked in previously, but whatever, current state is an improvement.

**Review by NotMyFault — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
